Src/BootCamp.Chapter/Program.cs

```diff
@@ -6,7 +6,24 @@ class Program
     {
         static void Main(string[] args)
         {
-            Console.WriteLine("Hello World!");
+            Console.WriteLine("Hello. Please state your name:");
+            string name = Console.ReadLine();
+
+            Console.WriteLine("Please state your age:");
+            int age = Convert.ToInt32(Console.ReadLine());
+
+            Console.WriteLine("Please state your weight:");
+            float weight = float.Parse(Console.ReadLine());
+
+            Console.WriteLine("Please state your height:");
+            float height = float.Parse(Console.ReadLine());
+
+            Console.WriteLine($"{name} is {age} years old, his weight is {weight} kg and his height is {height} cm. ");
+
+            //BMI calculations
+            double bmi = weight * 703 / height / height;
```

Comment:
Where does 703 come from? Is it a leftover from what you're supposed to read from a console?

Reply:
i found somewhere online that said thats the formula for calculating BMI.

Reply:
Don't trust random online sources and verify from multiple sources :)